react-stopwatch-component/my-app/src/Stopwatch.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+import { FaPlay, FaPause } from 'react-icons/fa6';
+
+export function Stopwatch() {
+  const [count, setCount] = useState(0);
+  const [isPaused, setIsPaused] = useState(false);
+
+  useEffect(() => {
```

Comment:
This is a nice solution. We haven't learned `useEffect` yet, so this can be solved without it. That solution is to start the interval timer when the Start icon is clicked instead of doing it in the useEffect.